cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
@bthebaudeau If you are using `fireflies` (the prototype A) keep in mind you need a pigtail to connect an external 2.4 GHz antenna.  This perhaps explains it.  Send me an email and we can check this